torch/csrc/jit/ir.h

```diff
@@ -934,11 +940,17 @@ struct Block {
       owning_node_; // either the node that has this block or nullptr for root
 };
 
+struct SourceGraph {
```

Comment:
what is this for?

Reply:
arrgh, will remove! `SourceGraph` was used in another version w/o profiling instructions.